app/connectors/des/DesConnector.scala

```diff
@@ -52,12 +52,14 @@ class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpCl
     httpClient.GET[VatObligations](getObligationsURL)
   }
 
-  def getFinancialData(vrn: String): Future[FinancialData] = {
+  def getFinancialData(vrn: String): Future[Option[FinancialData]] = {
     Logger.debug(s"Calling des api 1166 for vrn ${vrn}")
     implicit val hc: HeaderCarrier = desHeaderCarrier
     val getFinancialURL: String = s"$serviceURL$financialsUrl/${vrn}/VATC?onlyOpenItems=true"
     Logger.debug(s"""Calling des api 1166 with url ${getFinancialURL}""")
-    httpClient.GET[FinancialData](getFinancialURL)
+    httpClient.GET[Option[FinancialData]](getFinancialURL).recover {
+      case _: NotFoundException => None
```

Comment:
The recover is not needed. Please leave only:
```
httpClient.GET[Option[FinancialData]](getFinancialURL)
```

Reply:
Sure .. makes it nicer.  I originally did not have it as optional .  So was a hangover.